src/App.jsx

```diff
@@ -14,57 +16,65 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+export function App() {
+  const [selectedGood, setGood] = useState('Jam');
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  const isGoodSelected = good => good === selectedGood;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      {selectedGood !== null ? (
+        <h1 className="title is-flex is-align-items-center">
+          {selectedGood} is selected
+          <button
+            onClick={() => setGood(null)}
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
 
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+      <table className="table">
+        {goods.map(good => (
+          <tr
+            data-cy="Good"
+            className={
+              isGoodSelected(good) ? 'has-background-success-light' : ''
```

Comment:
use variable for it 
for this, create a variable in the middle of `map`
```suggestion
              isGoodSelected ? 'has-background-success-light' : ''
```
also u can use [classnames](https://www.npmjs.com/package/classnames) library for all such cases

Reply:
oh, i forgot to change it to 'cn', dont blame on me 😂